### ISSUE TITLE
[Suggestion] Add an 'Overview' section to the doc.

### ISSUE THREAD

**Issue by kryzar** [papis/papis#328]
#### Problem

Hi! Papis is obviously capable of doing many things. However, I feel like the documentation is not very helpful to give an overview of its capabilities and possible workflows. If you know exactly what command to use, then the documentation is helpful. But if your question is broad like "How do I browse my database?", "How do I open Silverman's book?", "How do I organise my database?", then guidance would be welcome.

#### Suggestion

Therefore, I suggest adding an _Overview_ to the documentation, with workflows and tasks examples. At the moment (last `doc/` commit is [8115f63](https://github.com/papis/papis/commit/8115f630a1ebe82dc8c913e7bd4d75f110308c13)), the gap between the `Quick start` section and the rest of the documentation is too big in my opinion.

I think this would tremendously help this project grow, as Papis is very clean and basically the only software in its category. It has the most important features, it just lacks maturity.

Let me know if I can help.

**Comment by alejandrogallo** [comment]
I agree in general, realistically I'll not be able to write much guides or documentation any time soon, but I'll try also on my part.
I had hoped that some people would also jump in and write about their workflows, but this has not really materialized yet.

**Comment by kryzar** [comment]
Ok, thanks for your answer! I don't plan to use Papis in the short term because I don't have the time to get around it, but when I do, I would be glad to help and write stuff. In that case I will come back to you. For the moment I plan to try stuff from time to time.

**Comment by alexfikl** [comment]
We have a basic overview and "getting started" section now. There's also a "user guides" section that gives examples on some specific workflows. It's not great, but a bit better. :grin:
